### PR TITLE
pr number is updated

### PR DESCRIPTION
**Pull request by swaroopa86**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PRL-1106

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: charts/prl-ccd-definitions/Chart.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ name: prl-ccd-definitions
 version: 0.0.13
 dependencies:
   - name: nodejs
-    version: ~2.4.8
+    version: ~2.4.13
```
Comment (yogendraU) [inline charts/prl-ccd-definitions/Chart.yaml:7]: can you please make it precisely 2.4.12?

File: charts/prl-ccd-definitions/values.preview.template.yaml [hunk 2]
```diff
@@ -123,7 +123,7 @@ ccd:
         OIDC_ISSUER: https://forgerock-am.service.core-compute-idam-aat.internal:8443/openam/oauth2/hmcts
         ELASTIC_SEARCH_ENABLED: true
         ELASTIC_SEARCH_HOST: ${SERVICE_NAME}-es-master
-        ELASTIC_SEARCH_CASE_INDEX_MAPPING_FIELDS_LIMIT: 15000
+        ELASTIC_SEARCH_CASE_INDEX_MAPPING_FIELDS_LIMIT: 16000
```
Comment (yogendraU) [inline charts/prl-ccd-definitions/values.preview.template.yaml:126]: is this change intended for this PR?

**Review by yogendraU — CHANGES_REQUESTED** [review]

File: charts/prl-ccd-definitions/Chart.yaml (hunk 1 above)
Comment (swaroopa86) [reply charts/prl-ccd-definitions/Chart.yaml:7]: Just for a trail run of the prs i have upgraded after it is failed for 2.4.12. I will revert this change

File: charts/prl-ccd-definitions/values.preview.template.yaml (hunk 2 above)
Comment (swaroopa86) [reply charts/prl-ccd-definitions/values.preview.template.yaml:126]: this limit change currently applicable only for this pr. But once this has been merged into Release-3-1, i am not sure whether we need to keep the same limit. Can you please confirm me on this.

**Review by khurshedk — APPROVED** [review]
